lldb/test/API/commands/process/attach/main.cpp

```diff
@@ -12,10 +12,13 @@ int main(int argc, char const *argv[]) {
     // Waiting to be attached by the debugger.
     temp = 0;
 
+    if (argc > 1 && argv[1][0] == 'q')
+      return 0;
+   
     while (temp < 30) {
-        std::this_thread::sleep_for(std::chrono::seconds(2)); // Waiting to be attached...
-        temp++;
-    }
+      std::this_thread::sleep_for(std::chrono::seconds(2)); // Waiting to be attached...
+      temp++;
+    } 
```

Comment:
I know you did not create this test in this PR but I wonder if there's a way we can handle this another way? Waiting on timers can be tricky from a testing perspective.

Reply:
I'm actually not using that feature of the test in my test at all.  I only use this main passing `q` as the first argument so it doesn't sleep, it just exits immediately.  I think the sleep is there to keep this from busy-waiting.  You could have it do that forever and then rely on a test failure to cause it to get torn down?  But I'd rather not change that since it's entirely unrelated to the test I added...

Reply:
That's fair, thanks for clarifying.